en/02_Developer_Guides/00_Model/12_Indexes.md

```diff
@@ -39,7 +40,7 @@ use SilverStripe\ORM\DataObject;
 
 class MyObject extends DataObject
 {
-    private static $indexes = [
+    private static array $indexes = [
```

Comment:
Unrelated - just updating the config references in this page to follow latest best practice